ifrs17/Report/ReportMutableScopes.ipynb

```diff
@@ -111,7 +111,7 @@
                 "\n    int Year => ReportingPeriod.Year;",
                 "\n    int Month => ReportingPeriod.Month;",
                 "\n    string ReportingNode { get; set; }",
-                "\n    string Scenario { get; set; }",
+                "\n    string Scenario { get; set; }  // TODO: Dropdown selection including All and Delta",
```

Comment:
This will change, but OK.
BTW, I would suggest we separate the filter Scenario (where the scn names are defined) from the visualization (where you can choose 1 scenario or some comparison view).

Reply:
Decided to postpone the separation of scenario and comparison visualization in a later PR.